xsensmt/XsensMT.cpp

```diff
@@ -108,13 +108,29 @@ bool XsensMT::open(yarp::os::Searchable &config)
         yWarning() << "xsensmt -  Parameter \"frame_name\" not set. Using the same value as \"sensor_name\" for this parameter.";
     }
 
+    yDebug() << "xsensmt - test - xsensmt_period, check xsensmt_period: " << config.check("xsensmt_period");
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is string: " << config.find("xsensmt_period").isString();
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is int: " << config.find("xsensmt_period").isInt32();
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is float: " << config.find("xsensmt_period").isFloat64();
+
+    if (config.check("xsensmt_period") && ( config.find("xsensmt_period").isInt32() || config.find("xsensmt_period").isFloat64()))
+    {
+        m_outputPeriod = config.find("xsensmt_period").asFloat64();
+    }
+    else
+    {
+        m_outputPeriod = 10; // 10ms
```

Comment:
The latest style for most YARP devices is to use seconds to represent periods, see for example https://www.yarp.it//v3.5/classControlBoard__nws__yarp.html#details . Even if some icub devices still use milliseconds, I would try to be consistent with the new devices and use seconds here.

Reply:
Ok now the expected period is in seconds https://github.com/robotology/yarp-device-xsensmt/pull/34/commits/28bdd1b1c310f84e74e5d67c6032cc3c6f8d07a6